pygmt/_version.py

```diff
@@ -109,7 +109,8 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
 
 
 def versions_from_parentdir(parentdir_prefix, root, verbose):
```

Comment:
The `_version.py` file was generated by [`versioneer`](https://github.com/python-versioneer/python-versioneer), and we should exclude this file. However, the latest docformatter v1.3.1 doesn't provide the `--exclude` option, although its master version does.

Reply:
Ideally we would switch from versioneer to setuptools-scm (see https://github.com/pydata/xarray/issues/2853), but haven't got around to doing this yet :slightly_smiling_face:

Reply:
Yes, I also don't like versioneer, which adds too many files to the project and is less maintained until recently.

Reply:
Let's open up a separate issue/PR to tackle the versioneer->setuptools-scm migration.

Reply:
OK. Then this PR is ready for review.

Reply:
We switched from versioneer to setuptools-scm in PR #695, so we don't need to wait for the `--exclude` feature in docformatter.